repo/overrides/default.nix

```diff
@@ -64,6 +64,11 @@ in
 			})
 		) opamPackages.camlp4;
 
+		conf-sqlite3 = overrideAll (impl: {
+			nativeBuildInputs = (impl.buildInputs or []) ++ [ pkgconfig ];
```

Comment:
I assume this should be `(impl.nativeBuildInputs or [])`. conf-sqlite3 already depends on conf-pkg-config, which should depend on `pkgconfig`. Is it not getting propagated?